flare/agent_zoo/simple_rl_agents.py

```diff
@@ -11,23 +11,30 @@ class SimpleRLAgent(Agent):
     policy or off-policy RL algorithms.
     """
 
-    def __init__(self, env, num_games):
+    def __init__(self, env, num_games, action_noise=None):
         super(SimpleRLAgent, self).__init__(env, num_games)
+        self.action_noise = action_noise
 
     def _cts_store_data(self, observations, actions, states, rewards):
         assert len(observations) == 1 and len(actions) == 1
         self._store_data(
             'RL',
             dict(
-                sensor=observations[0],
-                action=actions[0],
-                reward=[r / 100.0 for r in rewards]))
+                sensor=observations[0], action=actions[0], reward=rewards))
```

Comment:
If you really want to generalize the reward shaping, you should write an additional API to add into the base class Agent, instead of just directly changing the scaling to 1. I left the / 100.0 there as a temporary solution.